.yaml-lint.yml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
I copy-pasted this file from the super-linter templates, and adapted the max. line lenght